Makefile

```diff
@@ -7,13 +7,12 @@ build:
 redeploy-container:
 	docker-compose build --pull
 	docker-compose exec web rake db:migrate
-	docker-compose exec web bower install --allow-root
-	docker-compose exec web bower update --allow-root
 	docker-compose exec web rake assets:precompile
 	docker-compose exec web rake tmp:cache:clear
 	docker-compose down --remove-orphans
 	rm -f ./tmp/pids/server.pid
 	docker-compose up -d
+	docker-compose exec -T web yarn --ignore-engines --ignore-scripts --modules-folder public/lib 
```

Comment:
You've removed the javascript package installation from line 10/11 and put it into line 15. This effectively waits until *after* rebooting the Rails app to get going with Javascript installation. So if Javascript installation fails, you've already restarted the app possibly broken by missing Javascript dependencies. If you move this line to line 10, then if yarn fails the app is still running fine and no harm was done. If you agree please do so! Thanks!

Reply:
Ok I will move the yarn installation command to line 11. Thanks